control-announcements/src/main/java/org/oskari/announcements/helpers/AnnouncementsParser.java

```diff
@@ -0,0 +1,69 @@
+package org.oskari.announcements.helpers;
+
+import fi.nls.oskari.control.ActionParameters;
+import fi.nls.oskari.control.ActionParamsException;
+import org.oskari.announcements.helpers.AnnouncementParams;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Search helper
+ */
+public class AnnouncementsParser {
```

Comment:
Use this in controller to parse request -> announcement. Not in the "db helper".